R/facet-.r

```diff
@@ -433,7 +436,25 @@ eval_facet <- function(facet, data, env = emptyenv()) {
     return(out)
   }
 
-  eval_tidy(facet, data, env)
+  # clone the env in order to prevent side effects (hopefully)
+  env <- env_clone(quo_get_env(facet))
+
+  # create a env with active bindings
+  cushioning_env <- child_env(env_parent(env))
```

Comment:
From the documentation of `child_env()`:
> `child_env()` is in the questioning stage. It is redundant now that `env()` accepts parent environments.

Reply:
Oops, thanks!